scale/diagnostic/views.py

```diff
@@ -23,7 +23,16 @@ class QueueScaleBakeView(GenericAPIView):
     """This view is the endpoint for queuing new Scale Bake jobs."""
     parser_classes = (JSONParser,)
     queryset = Queue.objects.all()
-    serializer_class = QueueStatusSerializerV6
+    
+    def get_serializer_class(self):
+        """Returns the appropriate serializer based off the requests version of the REST API. """
+
+        if self.request.version == 'v6':
+            return QueueStatusSerializerV6
+        elif self.request.version == 'v7':
+            return QueueStatusSerializerV6
+        else:
+            return QueueStatusSerializer
```

Comment:
We should get rid of this else and remove the queuestatusserializer class

Reply:
Removed